docs/content/installation/nic-images/use-aws-image.md

```diff
@@ -82,8 +87,59 @@ Make sure you have an operational EKS cluster and that the namespace for your NG
 
     {{< img title="ECR pull instructions for NGINX Ingress Controller" src="./img/ecr-pull-instructions.png" >}}
 
+    {{< tip >}}For help with credentials, AWS Labs offers a credential helper. Check out [their GitHub repository](https://github.com/awslabs/amazon-ecr-credential-helper) for setup instructions.{{< /tip >}}
+
+
+4. Update the image in the _nginx-plus-ingress.yaml_ manifest.
+
+{{%/tab%}}
+
+{{%tab name="helm"%}}
+
+Make sure you have an operational EKS cluster and that the namespace for your NGINX Ingress Controller is set up. If you don't have an EKS cluster yet, you'll need to create one.
+
+1. Associate your EKS cluster with an OIDC IAM provider. Use your specific `--cluster <name`> and `--region <region>` values.
+
+    ``` shell
+    eksctl utils associate-iam-oidc-provider --region=us-east-1 --cluster=my-cluster --approve
+    ```
+
+2. Create an IAM role and a service account for your cluster. Replace `--name <name>`, `--namespace <name>`, `--region <region>`, `--cluster <name>` and `--role-name <name>` with your values.
+
+    ``` shell
+    eksctl create iamserviceaccount --name nginx-ingress --namespace nginx-ingress --cluster my-cluster --region us-east-1 --attach-policy-arn arn:aws:iam::aws:policy/AWSMarketplaceMeteringRegisterUsage --role-only --role-name my-cluster-sa --approve
+    ```
+
+    This step creates the IAM role with the required policy, which we will later refer to in the helm values. For additional details, consult the [AWS documentation](https://docs.aws.amazon.com/eks/latest/userguide/create-service-account-iam-policy-and-role.html).
+
     <br>
 
+    Ensure the service account name matches the one in your _values.yaml_ file for helm deployment.  
+    Ensure the EKS `role-arn` matches the service account annotation in your _values.yaml_ file for helm deployment.  You can use this command to retrieve the `role-arn`
+    ``` shell
+    aws iam list-roles | jq -r --arg role "my-cluster-sa" '.Roles[] | select(.RoleName==$role) | .Arn'
+    ```
+
+    Here's what a sample _values.yaml_ file might look like:
+
+    ``` yaml
+    controller:
+      nginxplus: true
+      image:
+        repository: 709825985650.dkr.ecr.us-east-1.amazonaws.com/nginx/nginx-plus-ingress
+        tag: "{{< nic-version >}}-mktpl"
+      serviceAccount:
+        annotations:
+          eks.amazonaws.com/role-arn: arn:aws:iam::0123456789:role/my-cluster-sa
+        name: nginx-ingress
+    ```
+
+3. Sign in to the AWS ECR registry that specified in the instructions on the [AWS Marketplace portal](https://aws.amazon.com/marketplace/pp/prodview-fx3faxl7zqeau?sr=0-1&ref_=beagle&applicationId=AWSMPContessa).
+
+    {{< img title="ECR pull instructions for NGINX Ingress Controller" src="./img/ecr-pull-instructions.png" >}}
+
     {{< tip >}}For help with credentials, AWS Labs offers a credential helper. Check out [their GitHub repository](https://github.com/awslabs/amazon-ecr-credential-helper) for setup instructions.{{< /tip >}}
```

Comment:
@pdabelf5 I'm not sure why tip isn't working within a tab (I tried debugging it locally), but I have a workaround that might be better UX overall.

Shift the tip callout to before the tab altogether - tabs are intended for divergent steps to accomplish a task using a different process or tools.

If the credential helper is useful in either scenario, it should be shown to the reader before the callout, since it applies either way. This might also apply to the image beforehand.

Reply:
I have moved the tip to the end of the file, thanks @ADubhlaoich